data/en/getlocalelanguage.json

```diff
@@ -7,10 +7,7 @@
 	"description":"Gets the language from where the locale belongs to.",
 	"params": [],
 	"engines": {
-		"coldfusion": {"minimum_version":"", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-e-g/getlocaledisplayname.html"},
-		"lucee": {"minimum_version":"", "deprecated":"5", "notes":"Lucee5+ Deprecated in favor of getLocaleInfo()", "docs":"http://docs.lucee.org/reference/functions/getlocaledisplayname.html"},
-		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/function/getlocaledisplayname"},
-		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/getlocaledisplayname"}
+		"lucee": {"minimum_version":"", "added": "5.0", "deprecated":"5.0", "notes":"Lucee5+ Deprecated in favor of getLocaleInfo()", "docs":"http://docs.lucee.org/reference/functions/getlocaledisplayname.html"},
```

Comment:
Need to remove the trailing comma here for it be valid JSON

Reply:
Oh, you're right.